docs/01-overview/02-main-areas/application-connectivity/ac-05-pass-access-token-in-request-header.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Pass an access token in a request header
```

Comment:
Isn't it also more like detailed info that should be in a different section?

Reply:
Moved to Operations.